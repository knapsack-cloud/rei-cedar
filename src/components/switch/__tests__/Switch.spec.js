import { shallowMount } from '../../../../test/vue-jest-style-workaround.js';
import CdrSwitch from '../CdrSwitch.vue';

describe('CdrSwitch', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(CdrSwitch);
    expect(wrapper.element).toMatchSnapshot();
  });
});
