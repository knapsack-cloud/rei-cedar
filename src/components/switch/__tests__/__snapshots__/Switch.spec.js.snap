// Vitest Snapshot v1

exports[`CdrSwitch > matches snapshot 1`] = `
<div
  class="cdr-switch cdr-switch--medium"
>
  <div
    class="cdr-switch__label"
  >
    
    
  </div>
  <button
    aria-checked="false"
    class="cdr-switch__button"
    role="switch"
  >
    <icon-check-sm-stub
      class="cdr-switch__button-icon"
    />
    <icon-x-sm-stub
      class="cdr-switch__button-icon"
    />
    <div
      class="cdr-switch__handle"
    />
  </button>
</div>
`;
